cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/apps/map.js

```diff
@@ -218,6 +218,9 @@ MortgagePerformanceMap.prototype.renderTooltip = function() {
   return ( point, meta ) => {
     var percent;
     var nationalPercent = Math.round( meta.national_average * 1000 ) / 10;
+    if ( point.value === null ) {
```

Comment:
@marteki The logic here isn't very intuitive. Both the county view and the metro view have map locations that need to show an "insufficient data" tooltip. But for metros we want to show the metro name and the national average along with an insufficient data message. For counties we only want to show the insufficient data message.

In metro view, the API returns a list of all metros and insufficient areas are set in cfpb-chart-builder as having a value of `-1` to indicate there's no data but we still need to show the location on the map.

In county view, the API return a list of *only* counties with sufficient data. So cfpb-chart-builder sets every pixel of the map that's not a county to have the value `null`.

This `renderTooltip` function is location agnostic and doesn't know whether we're in county or metro view so it just checks for `null` (which only appears in county view) or `< 0` (which only appears in metro view) and sets tooltips accordingly.

State view has no insufficient data.

In the future if we added a new location type (census tracts or something) the tooltip would work out of the box based on the data passed to it from the API.

Reply:
> But for metros we want to show the metro name and the national average along with an insufficient data message.

Confirmed with design that we don't want to show national average unless we have an actual average to compare it to. But changing that is out of the scope of this PR, maybe.